Modules/Tracker/QuestieTrackerPrivates.lua

```diff
@@ -8,56 +8,335 @@ local QuestieCombatQueue = QuestieLoader:ImportModule("QuestieCombatQueue")
+    -- setPoints so the frame doesn't appear to shift or "snap" from it's desired location when the
+    -- new setPoints are applied. Inputs are the frame, and desired setpoint.
+    -----------------------------------------------------------------------------------------------
+    if ({frame:GetPoint()})[2]:GetName() == "UIParent" and setpoint then
+        local height = frame:GetHeight()
+        local width = frame:GetWidth()
+        local setPoint = setpoint
+        local xOn = ({frame:GetPoint()})[4]
+        local yOn = ({frame:GetPoint()})[5]
+        if not (height or width or preSetPoint or xOn or yOn or setPoint or preSetPoint)then return end
+        if (setPoint ~= preSetPoint) then
+
+            -- To "TOPLEFT" from...
+            if setPoint == "TOPLEFT" then
+                if preSetPoint == "TOPRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn
+                elseif preSetPoint == "BOTTOMLEFT" then
+                    xOff = xOn
+                    yOff = yOn + height
+                elseif preSetPoint == "BOTTOMRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn + height
+                end
+                return xOff, yOff
+
+            -- To "TOPRIGHT" from...
+            elseif setPoint == "TOPRIGHT" then
+                if preSetPoint == "TOPLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn
+                elseif preSetPoint == "BOTTOMLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn + height
+                elseif preSetPoint == "BOTTOMRIGHT" then
+                    xOff = xOn
+                    yOff = yOn + height
+                end
+                return xOff, yOff
+
+            -- To "BOTTOMLEFT" from...
+            elseif setPoint == "BOTTOMLEFT" then
+                if preSetPoint == "TOPRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn + -height
+                elseif preSetPoint == "TOPLEFT" then
+                    xOff = xOn
+                    yOff = yOn + -height
+                elseif preSetPoint == "BOTTOMRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn
+                end
+                return xOff, yOff
+
+            -- To "BOTTOMRIGHT" from...
+            elseif setPoint == "BOTTOMRIGHT" then
+                if preSetPoint == "TOPRIGHT" then
+                    xOff = xOn
+                    yOff = yOn + -height
+                elseif preSetPoint == "TOPLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn + -height
+                elseif preSetPoint == "BOTTOMLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn
+                end
+                return xOff, yOff
+            end
+        else
+            return xOn, yOn
+        end
```

Comment:
Same as above: This should be split into multiple functions

Reply:
Fixed in latest commit